src/platforms/python/common/configuration/integrations/asyncpg.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: asyncpg
+description: "Learn about importing the asyncpg integration and how it captures queries from asyncpg."
+sidebar_order: 25
+---
+
+The asyncpg integration captures queries from
+[asyncpg](https://github.com/MagicStack/asyncpg) and shows them in the "Performance" seciton of Sentry.
+
+## Install
+
+Install `sentry-sdk` from PyPI.
+
+```bash
+pip install --upgrade sentry-sdk
```

Comment:
Do you plan to have the `asyncpg` integration installed per default, otherwise, this would need to change to 

```suggestion
pip install --upgrade sentry-sdk[asyncpg]
```